ocs_ci/ocs/node.py

```diff
@@ -1211,7 +1212,7 @@ def is_node_labeled(node_name, label=constants.OPERATOR_NODE_LABEL):
     return node_name in node_names_with_label
 
 
-def taint_nodes(nodes, taint_label=constants.OCS_TAINT):
+def taint_nodes(nodes, taint_label=None):
```

Comment:
I think you shoud change now also the docstring:
```
Default value is the OCS taint
```
to something like: If None (Default value) the constants.OPERATOR_NODE_TAINT will be used.